algo.typ

```diff
@@ -237,43 +231,35 @@
   line-number-styles,
 ) = { locate(loc => style(styles => {
   let id-str = str(counter(_algo-id-ckey).at(loc).at(0))
-  let line-index-str = str(line-index)
   let comment-dicts = _algo-comment-dicts.final(loc)
   let comment-content = comment-dicts.at(id-str, default: (:))
-                                     .at(line-index-str, default: [])
+                                     .at(str(line-index), default: [])
 
   // heuristically determine the height of the containing table row
   let row-height = calc.max(
     // height of main content
     measure(
-      {
-        set text(..main-text-styles)
-        _alphanumerics
-        content
-      },
+      main-text-styles(
+        _alphanumerics + content
+      ),
       styles
     ).height,
 
     // height of comment
     measure(
-      {
-        set text(..comment-styles)
-        comment-content
-      },
+      comment-styles(comment-content),
       styles
     ).height,
 
     // height of line numbers
     measure(
-      {
-        set text(..line-number-styles)
-        _numerals
-      },
+      line-number-styles(line-index),
       styles
     ).height
   )
 
-  // converting input parameters to absolute lengths
+  // TODO: WHY?!
```

Comment:
As far as I understand, taking the width directly has no different behaviour, or am I overseeing some weird special case?